typing/typeclass.ml

```diff
@@ -1583,7 +1600,7 @@ let final_decl env define_class
 
   (* make the dummy method disappear *)
   begin
-    let self_type = Ctype.self_type clty.cty_type in
+    let self_type = Ctype.self_type clty.cty_type in 
```

Comment:
```suggestion
    let self_type = Ctype.self_type clty.cty_type in
```
I'd recommend installing the Git `pre-commit` hook detailed in [CONTRIBUTING.md](https://github.com/ocaml/ocaml/blob/trunk/CONTRIBUTING.md#coding-guidelines) (I'm biased, as I wrote it!)

Reply:
(I use `./tools/check-typo-since trunk`. Is this all properly documented in HACKING.adoc?)

Reply:
@gasche coding guidelines and `tools/check-typo` are nicely documented in [CONTRIBUTING.md](https://github.com/ocaml/ocaml/blob/trunk/CONTRIBUTING.md#coding-guidelines). My bad not checking it out first.